server/lib/nl/page_config_builder.py

```diff
@@ -540,6 +543,7 @@ def _ranking_chart_block_pc(column, pri_place: Place, pri_sv: str,
   _set_ranking_tile_spec(attr['ranking_types'], pri_sv, tile.ranking_tile_spec)
   tile.title = _decorate_chart_title(title=sv2thing.name[pri_sv],
```

Comment:
nit: does https://github.com/datacommonsorg/website/blob/33a84f6c3b8c9fe0557907f442e0780f46af08e7/server/tests/lib/nl/page_config_builder_test.py#L377 not need fixing?

Reply:
ah yes- done.